extensions/amp-base-carousel/1.0/storybook/Basic.js

```diff
@@ -30,8 +30,9 @@ export const _default = () => {
   const height = number('height', 225);
   const slideCount = number('slide count', 5, {min: 0, max: 99});
   const colorIncrement = Math.floor(255 / (slideCount + 1));
+  const controls = select('show controls', ['auto', 'always', 'never']);
```

Comment:
Probably worth hoisting a const with this array up?

Reply:
Good point, done.